pisa/core/distribution_maker.py

```diff
@@ -430,27 +429,6 @@ def _set_rescaled_free_params(self, rvalues):
                         ' fixed in at least one pipeline' %name
                     )
 
-    def fit(self, data, metric, minimizer_settings, 
```

Comment:
why are we removing these?

Reply:
These caused an import loop when `pisa.analysis` was imported. Analysis imports core, but now core tries to import analysis, which at this point doesn't exist yet.

Reply:
I see.....ok, then we "simply" need to fix the import

Reply:
But this wasn't in the previous interface so no one is relying on these fit methods yet, no? At this point we can just remove it and not have any problems...